rfd/0090-db-mfa-sessions.md

```diff
@@ -0,0 +1,175 @@
+---
+author: Gavin Frazar (gavin.frazar@goteleport.com)
+state: draft
+---
+ 
+# RFD 90 - Database MFA Sessions
+
+## Required Approvers
+- Engineering: Marek `@smallinsky`, Roman `@r0mant`
+- Product: Sasha `@klizhentas`, Xin `@xinding33`
+
+## What
+When per-session-MFA is enabled, we should not restrict database cert TTL to 1 minute.
```

Comment:
This is a little too low-level of an opening sentence. IMHO, the real "what" is:

Prevent database access users from being required to tap an MFA device every minute when per-session-MFA is enabled.

Reply:
I agree - moving this low-level description under "Details" section and replacing with your suggestion.